pallets/utxo/src/lib.rs

```diff
@@ -573,6 +740,28 @@ pub mod pallet {
             Self::deposit_event(Event::<T>::TransactionSuccess(tx));
             Ok(().into())
         }
+
+        #[pallet::weight(T::WeightInfo::tokens_create(768_usize.saturating_add(token_name.len()) as u32))]
+        pub fn tokens_create(
+            origin: OriginFor<T>,
+            public: H256,
+            input_for_fee: TransactionInput,
+            token_name: String,
+            token_ticker: String,
+            supply: Value,
+        ) -> DispatchResultWithPostInfo {
+            let caller = &ensure_signed(origin)?;
+            tokens_create::<T>(
+                caller,
+                public,
+                input_for_fee,
+                token_name,
+                token_ticker,
+                supply,
+            )?;
+            Self::deposit_event(Event::<T>::TokenCreated(1u32, caller.clone()));
```

Comment:
what does the `1u32` denote?

Reply:
Another good catch, it should be TokenID

Reply:
Fixed